content/en/docs/reference/glossary/immutable-infrastructure.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Immutable Infrastructure
+id: immutable-infrastructure
+date: 2024-03-25
+full_link:
+short_description: >
+  Immutable Infrastructure refers to computer infrastructure (virtual machines, containers, network appliances) that cannot be changed once deployed
+
+aka: 
+tags:
+- architecture
+- storage
+- tool
```

Comment:
```suggestion
```
Immutable infrastructure is not a tool the way that `kubectl` and `kubeadm` are.